src/RegisterMappedEventSubscribers.php

```diff
@@ -41,14 +42,18 @@ public function __construct(Container $container)
      * @param Configuration $configuration
      * @param Connection    $connection
      * @param EventManager  $eventManager
+     *
+     * @throws ORMException
      */
     public function extend(Configuration $configuration, Connection $connection, EventManager $eventManager)
     {
+        $configRepository = \Illuminate\Container\Container::getInstance()->get(Repository::class);
```

Comment:
this change also doesn't make sense since you doing the same thing the other way